modules/vsphere-multiple-hosts.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/vsphere-hosts.adoc
+
+[id="vsphere-multiple-hosts_{context}"]
+= Configuring multiple vSphere hosts
+
+You can configure your cluster to run on multiple VMware vSphere hosts.
+
+.Prerequisites
+
+* You installed a {product-title} cluster on vSphere.
+* You reviewed the
+link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/existing.html[vSphere Cloud Provider Configuration]
+and can provide the data that is required for the `VirtualCenter` hosts that you
+add.
+
+.Procedure
+
+. Edit the `cloud-provider-config` ConfigMap:
++
+----
+$ oc edit configmap/cloud-provider-config -n openshift-config
+----
++
+----
+apiVersion: v1
+data:
+  config: |+
+    [Global]
+    secret-name      = vsphere-creds
+    secret-namespace = kube-system
+    insecure-flag    = 1
+
+    [Workspace]
+    server            = server.example.com
+    datacenter        = dc1
+    default-datastore = nvme-ds1
+    folder            = blah
+
+    [VirtualCenter "server.example.com"] <1>
+    datacenters = dc1
+
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: cloud-provider-config
+  namespace: openshift-config
+----
+<1> Add more `[VirtualCenter]` sections to define the additional hosts for your
+cluster. See the
+link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/existing.html[vSphere Cloud Provider Configuration]
```

Comment:
Per the vmware doc, should we also update 'datacenters' parameter in 'Global' section?

Reply:
Per vmware doc example, should be something like:
datacenters = "dc1, dc2"
Because I have no such multiple host vsphere cluster to validate it, actually I am not 100% sure my above statement is correct or not, just a question from me. Maybe need dev @staebler confirm.